server/http.go

```diff
@@ -24,15 +24,29 @@ import (
 )
 
 const (
-	defaultMeetingTopic      = "Zoom Meeting"
-	zoomOAuthUserStateLength = 4
+	DefaultMeetingTopic      = "Zoom Meeting"
+	ZoomOAuthUserStateLength = 4
+	SettingDataError         = "something went wrong while getting settings data"
+	AskForPMIMeeting         = "Would you like to use your personal meeting ID?"
+	APIToUpdatePMI           = "/plugins/%s/api/v1/updatePMI"
+	APIToAskForPMI           = "/plugins/%s/api/v1/askPMI"
+	Yes                      = "Yes"
+	No                       = "No"
+	Ask                      = "Ask"
+	Action                   = "action"
+	UserID                   = "userID"
+	ChannelID                = "channelID"
+	UsePersonalMeetingID     = "USE PERSONAL MEETING ID"
+	UseAUniqueMeetingID      = "USE A UNIQUE MEETING ID"
+	MattermostUserID         = "Mattermost-User-ID"
```

Comment:
Capitalized variables imply the value is meant to be consumed by another package, though I don't think that's the case for any of these constants